docs/concepts/policies/README.md

```diff
@@ -0,0 +1,96 @@
+## How to use policies with Express
+
+### Fastest Way : Use Yeoman
+
+```Bash
+  yo trails:policy ExamplePolicy
+```
+
+With Yeoman you don't need to create the file and include it in index.js
+
+### First Step : Write your policy
+
+The convention is that policies has to be written in api/policies.
+
+```JavaScript
+/**
+ * FILE: api/policies/Example.js
+ */
+'use strict'
+ const Policy = require('trails-policy')
+
+ /**
+  * @module ExamplePolicy
+  * @description This is the example policy
+  */
+ module.exports = class ExamplePolicy extends Policy {
+   test(req, res, next) {
+     const secret = req.body.secret
+     // Your own logic
+     /**
+     * The following code is an example:
+     * It stop when the secret parametter is passed and continue to the Controller when there is no secret parametter.
+     */
+     if (secret != '') {
+       // Stop and return with a custom message
+       return res.send('Your secret is' + secret)
+     }
+     // Continue to Example Controller
+     next()
+   }
+ }
+```
+### Second Step : Load your policy
+
+```JavaScript
+/**
+ * FILE: api/policies/index.js
+ */
+
+ 'use strict'
+
+ module.exports = {}
+ exports.Example = require('./Example')
+
+```
+
+### Third Step : Declare your policy in config the file
+
+```JavaScript
+/**
+ * FILE: config/policies.js
+ * The following Policies are example of what you can do with Trails Policies
+ */
+
+ module.exports = {
+
+   // Apply on every controllers
+   '*': ['ExamplePolicy.test'],
+
+   ExampleController: {
+     // Unique policy for one controller function
```

Comment:
Also `'*': ['ExamplePolicy.test2']` is allowed here to override the global `'*'`